timemachine/fe/bar.py

```diff
@@ -99,15 +99,37 @@ def dG_dw(w):
     return dG_dw
 
 
-def bootstrap_bar(w_F, w_R, n_bootstrap=1000, timeout=10) -> Tuple[float, NDArray]:
-    """Subsample w_F, w_R with replacement and re-run BAR many times
+def mbar_from_u_kln(u_kln: NDArray, **kwargs):
```

Comment:
weakly held style opinion: suspicious of forwarding **kwargs 

unsure if it's reasonable to make this more explicit (maybe expose only a solver tolerance option and initial guess option)?

Reply:
Could also make this just return `u_kln` and `N_k` and leave the construction of the `MBAR` instance up to the caller. I was on the fence about doing this.

Reply:
Should be addressed by https://github.com/proteneer/timemachine/pull/1098/commits/75da78a05c3929666ffa7c725f881ea34621aef0